sts/security-sts-server/src/main/java/ddf/security/sts/PropertyPlaceholderWrapper.java

```diff
@@ -45,10 +43,15 @@ public PropertyPlaceholderWrapper(DefaultConditionsProvider conditionsProvider,
             StaticSTSProperties properties) {
         samlConditionsProvider = conditionsProvider;
         stsProperties = properties;
+        // set the default values in case there is no configuration
+        samlConditionsProvider.setLifetime(1800);
```

Comment:
The way that it should work is that the constructor initializes the default values on bundle startup. If there are any existing configuration values, then the config handler method would get called immediately afterwards they would be updated with the correct values.

I suppose we could add the config to the constructor as well, but then we'd have to deal with some more null checking and conditional logic. That seems unnecessary to me, but could be done if you think it necessary.